src/corecel/sys/Device.cc

```diff
@@ -222,22 +223,6 @@ Device::Device(int id) : id_{id}, streams_{new detail::StreamStorage{}}
     // See device_runtime_api.h
     eu_per_cu_ = CELER_EU_PER_CU;
```

Comment:
Since we're updating launch bounds and with the recent fix in #968, where do we get the fact that CUDA has one execution unit per compute unit? According to the [definition](https://rocm.docs.amd.com/projects/HIP/en/latest/reference/kernel_language.html#cu-and-eu-definitions) `Compute Unit == Streaming multiprocessor` and `Execution Unit == Warp Scheduler`. Almost every single cuda architecture (except 6.0) have [4 warp scheduler per SM](https://docs.nvidia.com/cuda/cuda-c-programming-guide/#architecture-8-x) and warps are statically bound to one scheduler until completion.

Reply:
Oh yeah, we'd talked about this. I think I based "EU=CU" from the HIP launch bounds definition and some AMD documentation. Would you mind fixing this definition in a new PR? Then maybe our second-argument launch bound can have a meaning that more directly translates to "occupancy".

Reply:
sure, I'll have a look at it.